.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 docs/html_docs
 /html_docs
+launch.json
```

Comment:
This was added during debugging and should be kept to ignore debug configurations.